### DISCUSSION TITLE
Hint error

### DISCUSSION THREAD

**Discussion by Alerinos** [community/community#19359] [Copilot Conversations]
https://gyazo.com/6416be86a153e3f26d5f880c17ffae86
I have a very annoying bug. During the hint, I do not see it or my code. I don't know what's going on so I can't use copilot. It is very annoying when I do not see the code that I am writing every now and then. I want to delete the copilot.

**Comment by melvincwng** [comment]
@Alerinos, for bugs, you can report it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

Thanks and have a nice day ahead 😊

**Comment by JaimeTR** [comment]
@Alerinos Reinstall Github Copilot, that fixes it for me, or if possible install GitHub Copilot Nightly.

If this answer is helpful, please 'Mark as answer'. :)

**Reply by Alerinos** [reply]
I can't find this anywhere for visual studio. Can I have a link please?

**Reply by JaimeTR** [reply]
@Alerinos   https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

**Reply by Alerinos** [reply]
> @Alerinos https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

This is visual studio code, I use visual studio 2022 😄

**Reply by JaimeTR** [reply]
It only works for visual studio code
